css/src/components/gradient.scss

```diff
@@ -88,3 +88,15 @@ $border-gradient-direction: (
 	-webkit-text-fill-color: transparent;
 	line-height: 1.4; // required because of background clip, avoid "g" and "y" getting cut off
 }
+
+.gradient-card {
+	border: $border-width-md solid $border !important;
+	border-radius: $border-radius-lg !important;
+	border-color: transparent !important;
+	background: linear-gradient($body-background 0 0) padding-box,
```

Comment:
can `linear-gradient($body-background 0 0) padding-box,` be removed in favor off applying `background-color: $body-background;` on the -content element?